docs/includes/extracts-option-requires.yaml

```diff
@@ -1,48 +1,43 @@
 ---
 ref: option-requires-4.2
-source:
-  ref: option-requires-version
-  file: extracts-option-requires.yaml
```

Comment:
I was under the impression that both extracts and replacements were supported in Snooty. See slide 49 of [Marking up: Tech Writer Training on rST and YAML](https://docs.google.com/presentation/d/125-MvuDr66EIUgb3bVWCX3Uh2Jj2MkM8kM0iM_JoOuA/edit). Is this not the case, or is there some other limitation here (e.g. a file referencing itself)?

If you're going to remove the inheritance here, I see no reason to preserve the `replacement` block. The version number might as well just be hard-coded into the content. But before doing that, I would like to address my first question.

Reply:
Ok I switched it back to using inheritance - I changed "source:" to "inherit:" and that fixed the build issues. I think that rST / YAML powerpoint might need to be updated, I can investigate why "inherit" fixes the problem.

Reply:
Never mind, seems like the issue was the missing _ before "option-requires-version"